src/player/Player.php

```diff
@@ -2526,6 +2530,24 @@ public function getCraftingGrid() : CraftingGrid{
 		return $this->craftingGrid;
 	}
 
+	/**
+	 * Returns the default CreativeInventory instance if no custom one was set.
+	 */
+	public function getCreativeInventory() : CreativeInventory{
+		return $this->creativeInventory;
+	}
+
+	/**
+	 * To set a custom creative inventory, you need to make a clone of a CreativeInventory instance.
+	 * Make sure to use this method only in or after `PlayerJoinEvent` to sync the contents on the client.
```

Comment:
This doesn't seem appropriate. It will work in PlayerLoginEvent also (since that takes place before the network session sends the creative inventory initially.